cardinality/cardinality.go

```diff
@@ -87,6 +93,10 @@ func (promInstance *PrometheusCardinalityInstance) FetchTSDBStatus(prometheusCli
 	}
 	defer res.Body.Close()
 
+	if res.StatusCode == 401 {
+		return fmt.Errorf("401 Unauthorized. The provided Authorization value (%s) is incorrect.", promInstance.AuthValue)
```

Comment:
You shouldn't be logging the AuthValue as it's a credential. Just log that it's incorrect

Reply:
Removed this